tracer/build/_build/Build.Steps.cs

```diff
@@ -552,6 +552,73 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
             }
         });
 
+    Target DownloadLibDatadog => _ => _
+                .Unlisted()
+                .After(CreateRequiredDirectories)
+                .Executes(async () =>
+                {
+                    if (IsLinux || IsOsx)
+                    {
+                        if (!Directory.Exists(NativeBuildDirectory))
+                        {
+                            CMake.Value(
+                                arguments: $"-DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang -B {NativeBuildDirectory} -S {RootDirectory}");
+                        }
+                        // In CI, the folder might already exist. Which means that CMake was already configured
+                        // and libdatadog artifact (correct one) was already downloaded.
+                        // So just reuse it.
+
+                    }
+                    else if (IsWin)
```

Comment:
It looks like this branch always executes, regardless if it's already downloaded. Is that the case? Does it _need_ to?